src/modules/flight_mode_manager/tasks/AutoFollowTarget/FlightTaskAutoFollowTarget.hpp

```diff
@@ -72,26 +70,38 @@ static constexpr float ALT_ACCEPTANCE_THRESHOLD = 3.0f;
 // is too close to the ground (below MINIMUM_SAFETY_ALTITUDE)
 static constexpr float EMERGENCY_ASCENT_SPEED = 0.2f;
```

Comment:
is this a bit slow? what if velocity tracking is a bit bad? may not even go up. this is also after all an *emergency* ascent. @potaito, wdyt?

Reply:
Updated to 0.5!